lib/dialects/abstract/query-generator.js

```diff
@@ -234,7 +247,12 @@ class QueryGenerator {
       ].join(' ');
     }
 
-    return _.template(query, this._templateSettings)(replacements);
+    query = _.template(query, this._templateSettings)(replacements);
+    // Used by Postgres upsertQuery and calls to here with options.exception set to true
+    if (options.bindParam === false) {
+      return query;
```

Comment:
return an object in both cases, so all calls can just use `.query`. In case `options.bindParam: false` return no bind parameters.

Reply:
Done